src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -1420,17 +1420,26 @@ internal void LoadNodeLibrary(Assembly assem, bool suppressZeroTouchLibraryLoad
            // I think this is consistent with the current behavior - IE today - if a nodeModel exists in an assembly, the rest of the assembly 
            // is not imported as ZT - the same will be true if the assembly contains a NodeViewCustomization.
 
-           // TODO(mjk) draw up matrix of current behaviors which nodeLib flag can control.
-            if (!NodeModelAssemblyLoader.ContainsNodeModelSubType(assem)
-                && !(NodeModelAssemblyLoader.ContainsNodeViewCustomizationType(assem)))
+            bool nodeModelOrNodeView;
+            try
+            {
+                nodeModelOrNodeView = !NodeModelAssemblyLoader.ContainsNodeModelSubType(assem)
```

Comment:
shouldn't this variable be name be inverted `notNodeModelOrNodeView`  or `DoesNotContain_NodeModelOrNodeView`

Reply:
Same question as Mike.

Reply:
I kept the name and reversed the condition.